gridlab-d/gridlab-d
# Fix build error on macOS relating to problems with node::defaults

#### What's this Pull Request do?

The PR fixes #1134, a report of build errors on macOS related to node::defaults

#### Where should the reviewer start?

Best thing is to read #1134.

#### How should this be tested?

The problem is intermittent on macOS and seems to have something to do with a recent update to the systems being pushed by Apple.  To know for sure if it's fixed, you have to find a system on which the problem occurs prior to the fix and then check that it no longer occurs after the fix.  Good luck with that.

#### Any background context you want to provide?

The issue centers on the use of C++ API in modules that do not support the use of "defaults" for fast object creation.

#### What are the relevant issues?

Fast object creation is a legacy object creation strategy to support object arrays.  It's almost never used (in fact it may no longer be used at but it's hard to be sure).  The C++ API supported it but probably shouldn't because it's not safe to use with classes that have virtual function tables.

#### Screenshots (if appropriate)

N/A

#### Questions:
